setup.py

```diff
@@ -800,7 +800,14 @@ def determine_sdl2():
             'lib/libtess2/Source/tess.c'
         ]
     }),
-    'graphics/svg.pyx': merge(base_flags, gl_flags_base)
+    'graphics/svg.pyx': merge(base_flags, gl_flags_base),
+    'lang/builder.py': {},
+    'lang/parser.py': {},
+    'uix/videoplayer.py': {},
+    'uix/relativelayout.py': {},
+    'uix/floatlayout.py': {},
+    'uix/layout.py': {},
+    'uix/carousel.py': {},
```

Comment:
It looks like you cherry picked some layouts to cythonize, but not all of them, like boxlayout or anchorlayout?

Reply:
Yes, it was mostly opportunistic, i don't think there is much preventing adding others.

Reply:
This is a "dumb" cythonization, the kind that can sometime gets you a 300% performance increase, but rarely more, a more involved PR would make use of cython-specific syntax and make it mandatory to build these modules for usage, and potentially increase them much more.